processor/src/fast/execution_tracer.rs

```diff
@@ -603,12 +603,12 @@ impl Tracer for ExecutionTracer {
     }
 
     fn increment_clk(&mut self) {
-        // do nothing
+        self.overflow_table.advance_clock();
```

Comment:
Was this a bug previously?

Reply:
The API changed a couple of times, with one change before the last rebase. I have fixed this in https://github.com/0xMiden/miden-vm/pull/2472/commits/f2dac9e449af1dee223f18abffb4a9758b8441ce